components/Imprint/Employees.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react'
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+import { css } from 'glamor'
+
+import Employee from './Employee'
+import Loader from '../Loader'
+import {
+  Interaction,
+  mediaQueries
+} from '@project-r/styleguide'
+
+const { H2, H3 } = Interaction
+
+const clearfixStyle = {
+  // Micro clearfix hack.
+  '&::before': {
+    content: ' ',
+    display: 'table'
+  },
+  '&::after': {
+    content: ' ',
+    display: 'table',
+    clear: 'both'
+  }
+}
+
+const styles = {
+  container: css({
+    marginBottom: 30,
+    [mediaQueries.mUp]: {
+      marginBottom: 50
+    }
+  }),
+  group: css({
+    ...clearfixStyle,
+    marginTop: 30,
+    [mediaQueries.mUp]: {
+      marginTop: 50
+    }
+  }),
+  subgroup: css({
+    ...clearfixStyle,
+    marginTop: 10,
+    [mediaQueries.mUp]: {
+      marginTop: 20
+    }
+  }),
+  groupHeading: css({
+    marginBottom: 10,
+    [mediaQueries.mUp]: {
+      marginBottom: 20
+    }
+  }),
+  subgroupHeading: css({
+    margin: '5px 0',
+    [mediaQueries.mUp]: {
+      margin: '10px 0'
+    }
+  })
+}
+
+const getEmployeesGsheet = gql`
+  query {
+    employees {
+      userId
+      group
+      subgroup
+      name
+      title
+    }
+  }
+`
+
+const getEmployeesData = gql`
+  query getEmployeesData($ids: [ID!]) {
+    search(filter: { type: User, ids: $ids }) {
+      nodes {
+        entity {
+          ... on User {
+            id
+            portrait
+            username
+          }
+        }
+      }
+    }
+  }
+`
+
+const groupBy = (array, key) => {
```

Comment:
I propose to use [`nest`](https://github.com/d3/d3-collection#nests) from `d3-collection` for grouping lists – it's more succinct and less error-prone.